service/config.go

```diff
@@ -48,6 +48,9 @@ type Config struct {
 
 	// ResolverConfig is config for resolving registries.
 	ResolverConfig `toml:"resolver"`
+
+	// MinLayerSize skips remote mounting of smaller layers
+	MinLayerSize int `toml:"min_layer_size"`
```

Comment:
#345 adds `SnapshotterConfig` to `fs/config`. This field should be part of that.

np: How about renaming it to something like `MinLayerSizeOverride`, to distinguish it from the `MinLayerSize` in the CLI?

Reply:
Heck might as well call it `MinLayerSizeConfigOverride` no reason not to be explicit

Reply:
SnapshotterConfig is used for different purposes than snapshotterConfig. I think it would need to be in both. I'd be moving it out of the snapshotter struct, not out of the config for the main app.

Reply:
The CLI min layer size is for creating, not preparing/mouinting. This new option is not an override of that option, it applies at a different point in the workflow. If you use both, the larger one overrides the smaller, in either order.